content/blog/2023/when-to-choose-a-native-mobile-app.md

```diff
@@ -0,0 +1,174 @@
+---
+title: When to Choose a Native Mobile App
+sub: Part 2 -- Responsive Web App vs PWA vs Native Mobile App
+author: sondra
+date: 2023-06-07
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part2-native.jpg
+  alt: A stand of smartphones
+summary: |
+  If you have an idea for a digital product,
+  you may be wondering if you should build a
+  responsive web app, PWA, or native mobile app.
+  Is one option inherently better?
+  What are the pros and cons?
+  This is part 2 of a three-part series
+  unpacking the reasons to choose
+  one platform over another.
+  Let's dive into the native mobile app.
+---
+{% import 'embed.macros.njk' as embed %}
+
+If you haven't read part 1 yet, it has a great introduction to each option along
+with a quick comparison chart.
+
+**Read all three parts to compare app types:**
+
+1. [When to Choose a Responsive Web App](/2023/05/05/when-to-choose-a-responsive-web-app/)
+2. [When to Choose a Native Mobile App](/2023/06/07/when-to-choose-a-native-mobile-app/)
+3. [When to Choose a Progressive Web App](/2023/07/05/when-to-choose-a-progressive-web-app/)
+
+## Native Mobile App -- Best for Advanced Capabilities
+
+> **TL;DR**
+>
+> If your top priority is to provide a high level of specialized
+> capability, especially for a specific mobile device like Android
+> or iPhone, a native mobile app is likely the best option for
+> building your digital product.
+
+A native mobile app is an app created specifically for one particular mobile
+platform such as iOS, Android, or Windows. Native mobile apps are built using
+programming languages and tools unique to each platform. Swift or Objective-C
+for iOS. Kotlin, Java, and C++ languages along with SDK and APK tools for
+Android.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/yes.png',
+    alt: 'The word "yes" with a smiley and sparkles'
+  }],
+  attrs={
+    'style': '--column: content-start'
+  }
+) }}
+
+## Choose a Native Mobile App when…
+
+### Advanced Capabilities
+*…when your project requires a high level of hardware or software integration.*
```

Comment:
@jgerigmeyer @stacyk I removed the duplicated "when."

I'm struggling to find a good way to make this structure easier to follow because I really want to use the language of 'choose an X when,' but I also don't want the h3s to be long sentences that are hard to skim.

I could change the h2s to: "When to Choose an X"
I could keep the h3s the same.
And I could change the italicized phrases to: "Choose an X when it's important to keep initial development costs low."
At least that solves the problem of having to remember the set up for the phrase, but is there too much repetition?